docs/wiki/configurator/power-tab.md

```diff
@@ -73,7 +73,7 @@ Multiplier: 1 = this is to fine-tune the outcome of the calculation, a value of
 Settings for the amperage meter to measure the current draw correctly, using this formula:
 
 $$
-Current(Amps) = \frac\{ADC (mV)}\{amperage\_meter\_scale \times 10 + \frac\{amperage\_meter\_offset}\{1000}}
+Current(Amps) = (\frac{ADC (mV) * 10000}{amperage\_meter\_scale} + amperage\_meter\_offset) / 1000
```

Comment:
You can also write it like this
![grafik](https://github.com/user-attachments/assets/9b23f371-91f2-41a1-bcdc-550bcb24d1fd)
```suggestion
\text{Current}[A] = \frac{\text{ADC}[mV] \cdot 10}{\text{amperage_meter_scale}} + \frac{\text{amperage_meter_offset}}{1000}
```

Reply:
Or alternatively, like this
![grafik](https://github.com/user-attachments/assets/13c4d70e-d3fa-4c28-a721-2ca3f93a14b9)
```latex
\text{Current}[A] = \left( \frac{\text{ADC}[mV] \cdot 10000}{\text{amperage_meter_scale}} + \text{amperage_meter_offset} \right) \cdot \frac{1}{1000}
```

Reply:
Thanks. All formula's including I'd proposed (which was based on code) have same outcome. But your first proposal looks the best option.

Reply:
Hmm first suggestion does not seem to be correct syntax.

Reply:
Both are equivalent

Reply:
Should be resolved in last commit

Reply:
```
\text{Current}[A] = \frac{\text{ADC}[mV] \cdot 10}{\text{amperage_meter_scale}} + \frac{\text{amperage_meter_offset}}{1000}
```
vs
```
\text{Current[A] = } \frac{\text{ADC[mV]} \cdot 10}{\text{amperage\_meter\_scale}} + \frac{\text{amperage\_meter\_offset}}{1000}
```